### PR TITLE
Add procfile

### PR DESCRIPTION
**Pull request by andrewdotn**

### CODE REVIEW

File: scripts/dev-bootstrap
Comment (eddieantonio) [inline scripts/dev-bootstrap:7]: hey dawg i heard you like `make` 😂

File: scripts/dev-bootstrap
Comment (eddieantonio) [inline scripts/dev-bootstrap:14]: If you _really_ feel like it, I'm sure we could infer these lists from what files are called `*-manage.py` then use some [expansion magic](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) to get rid of the `-manage.py` stuff.

But I think this script will be so seldom used that it's not worth the confusing bashisms!

**Review by eddieantonio — APPROVED** [review]
LGTM! Thanks for making this and documenting it! I think @nienna73 should have a look at this before it's merged, because it incurs a workflow change.

**Review by nienna73 — APPROVED** [review]
The docs do a great job of explaining this change!

File: scripts/dev-bootstrap
Comment (andrewdotn) [reply scripts/dev-bootstrap:7]: Ha, did not think of that. I’ve added a comment mentioning it could be turned into a small Makefile.

File: scripts/dev-bootstrap
Comment (andrewdotn) [reply scripts/dev-bootstrap:14]: Oh, since all the script does is call the manage scripts it wouldn’t even need expansions. That said, as more stuff gets supported for these languages, I expect I will need to replace this script with something more intelligent before we get around to adding another language.

**Pull request merged**
